data/strings/strings.txt

```diff
@@ -31655,53 +31655,6 @@
     zh-Hans = 授予权限
     zh-Hant = 授予權限
 
-  [aa_connected_to_car_notification_title]
```

Comment:
Why it is here?

Reply:
It was an unused string and regeneration validation would fail on it. Its removed now in a different PR.